src/dll.py

```diff
@@ -1,125 +1,87 @@
-"""Doubly Linked List Data Structure."""
+"""Doubly linked list in python."""
+
+from linked_list import LinkedList
 
 
 class Node(object):
-    """Create a Node class."""
+    """A node for a doubly linked list."""
 
-    def __init__(self, data, next_data=None, previous_data=None):
-        """Initialize a Node instance."""
+    def __init__(self, data, next, prev):
+        """Build node attributes."""
         self.data = data
-        self.next = next_data
-        self.previous = previous_data
+        self.next = next
+        self.prev = prev
 
 
 class DoubleLinkedList(object):
-    """Create a doubly linked list class."""
+    """A doubly linked list."""
 
-    def __init__(self):
-        """Initialize a double link list instance."""
+    def __init__(self, iterable=None):
```

Comment:
The spec specifically says that this structure does not take an iterable on initialization